packages/test/k6/grafana-datasource.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: 1
```

Comment:
Is this file needed yet, since we don't have the Grafana integration?

Reply:
So far it is not needed because we can only execute them locally and see the results locally. But the contents of it  may be be useful if we cannot integrate it with Prometheus. That's the reason of keeping it